app/src/main/res/values-nb/strings-nb.xml

```diff
@@ -1,501 +1,830 @@
+    <string name="reset">Tilbakestill</string>
+    <string name="pebble_and_android_wear_options">Pebble og Android Wear alternativer</string>
+    <string name="customize_colours">Tilpass farger</string>
+    <string name="xdrip_plus_color_settings">xDrip+ fargeinnstillinger</string>
+    <string name="glucose_values_and_lines">Glukoseverdier og linjer</string>
+    <string name="high_glucose_values">Høy glukoseverdier</string>
+    <string name="in_range_glucose_values">Vanlige glukoseverdier</string>
+    <string name="default_color_selected">Standardfarge Valgt</string>
+    <string name="xdrip_plus_extra_settings">xDrip+ ekstra innstillinger</string>
+    <string name="example_chart">Eksempel Plott</string>
+    <string name="treatments_prediction_curves">Behandlinger / prediksjon kurver</string>
+    <string name="carbs_per_unit">Karbohydrater per Enhet</string>
+    <string name="everyday">Hver dag</string>
+    <string name="insulin_sensitivity_glucose_drop_per_unit">Insulin Sensitivitet: glukose reduseres per enhet</string>
+    <string name="transmitter_battery">batteriet i senderen</string>
+    <string name="very_low">VELDIG LAV</string>
+    <string name="low">lav</string>
+    <string name="space_units">" enheter"</string>
+    <string name="units">enheter</string>
+    <string name="carbs">karbohydrater</string>
+    <string name="when">når</string>
+    <string name="speak_your_treatment">Snakk inn behandlingen f.eks:\nx.x enheter insulin / xx gram karbohydrater</string>
+    <string name="speech_recognition_is_not_supported">Talegjenkjenning er ikke støttet</string>
+    <string name="treatment_note">Behandlingnotat</string>
+    <string name="could_not_export_database">Kunne ikke eksportere Database :(</string>
+    <string name="exported_to">"Eksportert til "</string>
+    <string name="default_to_voice_input_next_time">Bruk talegjenkjenning neste gang</string>
+    <string name="done">Ferdig</string>
+    <string name="entering_a_calibration_now_will_override_your_previously_entered_calibration">Legges det inn en kalibrering nå, vil det overstyre tidligere kalibrering.</string>
+    <string name="enter_blood_glucose_value">Skriv inn blodsukkerverdi</string>
+    <string name="this_does_not_get_used_for_anything_its_just_to_keep_track_of_differences_for_algorithem_improvement_purposes">Dette blir ikke brukt til noe, det er bare å holde styr på forskjellen mellom algoritmene for forbedringsformål</string>
+    <string name="no_active_alert_exists">Ingen aktiv varsling finnes</string>
+    <string name="active_alert_exists_named">Aktiv varsling eksisterer med navn</string>
+    <string name="bracket_not_snoozed">(Ikke utsatt)</string>
+    <string name="all_alerts_disabled_until">"Alle varsler deaktivert inntil "</string>
+    <string name="low_alerts_disabled_until">"Lave varsler deaktivert inntil "</string>
+    <string name="high_alerts_disabled_until"
```

Comment:
"Vis 8-timers gjennomsnittslinje" (drop the trailing "n")

Reply:
Line 523. Done